### PR TITLE
feat: support Schema link in url

### PR DESCRIPTION
**Pull request by zmitton**
i went with parameter name`schemaUrl`. Using the variable name `url` for _part_ of the total url string inside the url string seemed confusing.

#### example
`http://localhost:3000/?schemaUrl=https://raw.githubusercontent.com/etclabscore/open-rpc-ethereum-js/zac1/openrpc.json`

### CODE REVIEW

**Comment by zmitton** [comment]
fixes #35

File: src/App.js [hunk 1]
```diff
@@ -76,9 +76,25 @@ export default class App extends React.Component {
...
+    let urlParams = this.parseUrlParams()
+    if (urlParams['schemaUrl']) { 
+      console.log("SSSS", urlParams['schemaUrl'])
+      this._handleUrlChange(urlParams['schemaUrl'])
+    }
     setTimeout(this.refreshEditorData, 1000);
   }
 
+  parseUrlParams(){
+    let url = window.location.href
```
Comment (BelfordZ) [inline src/App.js]: Should use `window.location.search`
And use this to parse the url: https://www.npmjs.com/package/qs ... 20mil downloads per week.

File: src/App.js [hunk 2]
```diff
@@ -76,9 +76,25 @@ export default class App extends React.Component {
   }
 
   async componentDidMount() {
+    let urlParams = this.parseUrlParams()
+    if (urlParams['schemaUrl']) { 
+      console.log("SSSS", urlParams['schemaUrl'])
```
Comment (BelfordZ) [inline src/App.js]: xxx

**Comment by zmitton** [comment]
back to you

**Pull request merged**